runner/src/check_build_licenses.rs

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use super::*;
```

Comment:
What does this do?

Reply:
it just imports everything from the parent mod; see https://doc.rust-lang.org/stable/rust-by-example/mod/super.html

Reply:
In this specific case, which is the parent mod? 

Asking since here it's at the top of the file, whereas the rust doc mentions using it in a nested scope. Is it where this is called from?

Reply:
This mod is not a top level one. The top-level one is the one in `main.rs` or `lib.rs`, then each file creates its own scope based on the file name. So the fully qualified name of a type T in this file is really `::check_build_licenses::T`. `super::` gets back to the parent scope, which is the on in `main.rs` in this case.

Reply:
Makes sense, thanks for explaining :)